internal/super/output.go

```diff
@@ -0,0 +1,152 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package super
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"golang.org/x/exp/maps"
+	"math/rand"
+	"os"
+	sysExec "os/exec"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/onflow/flow-cli/pkg/flowkit/output"
+	flowkitProject "github.com/onflow/flow-cli/pkg/flowkit/project"
+	"github.com/onflow/flow-cli/pkg/flowkit/services"
+)
+
+func printDeployment(deployed []*flowkitProject.Contract, err error, contractPathNames map[string]string) {
+	clearScreen()
+	fmt.Println(helpBanner())
+
+	if err != nil {
+		fmt.Println(errorBanner())
+		fmt.Println(failureDeployment(err, contractPathNames))
+		return
+	}
+
+	fmt.Println(okBanner())
+	fmt.Println(successfulDeployment(deployed))
+}
+
+func successfulDeployment(deployed []*flowkitProject.Contract) string {
+	var out bytes.Buffer
+	okFaces := []string{"😎", "😲", "😱", "😜"}
```

Comment:
This is cool :). But why is the third one considered an ok-face :)?

Also can `GetRandomOKFace` be its own function somewhere. Sounds useful.

Reply:
haha yeah I might need to update emojis to be more consistently OK

Reply:
Probably the first time when emojis are discussed in PR review :D